pages/project-version/downloads/views/index.jsx

```diff
@@ -8,15 +9,16 @@ const ProjectDownloads = () => {
 
   const thisVersionIsGranted = project.granted && project.granted.id === version.id;
   const licenceStatus = thisVersionIsGranted ? project.status : 'inactive';
+  const title = get(version, 'data.title', 'Untitled project');
```

Comment:
This isn't exactly equivalent because it will allow titles of `''`. Still probably want to have an `|| 'Untitled project'`